### PR TITLE
capstone: actually fix pkg-config references

### PR DESCRIPTION
**Pull request by Tyilo**
See #36871 and #36872

### CODE REVIEW

**Comment by DomT4** [comment]
Is this necessary in your testing? Most ` pkg-config ` files don't include the ` include/'blah" ` directory, because most things are configured to search for ` prefix/include/toolname ` anyway. Is this not the case with things that look for Capstone?

OpenSSL for example:

```
prefix=/usr/local/Cellar/openssl/1.0.2
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include
```

Fontforge:

```
prefix=/usr/local/Cellar/fontforge/20141230
exec_prefix=${prefix}
bindir=${exec_prefix}/bin
libdir=${exec_prefix}/lib
includedir=${prefix}/include
datarootdir=${prefix}/share
mandir=${datarootdir}/man
localedir=${datarootdir}/locale
```

**Comment by jacknagel** [comment]
If upstream generates a pkg-config file with `include/capstone` in it, then we should mirror that, I think.

File: Library/Formula/capstone.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def install
     # As per the above inreplace, the pkgconfig file needs fixing as well.
     inreplace lib/"pkgconfig/capstone.pc" do |s|
       s.gsub! "/usr/lib", lib
-      s.gsub! "/usr/include/capstone", include
+      s.gsub! "/usr/include/capstone", include + "/capstone"
```
Comment (jacknagel) [inline Library/Formula/capstone.rb]: `"#{include}/capstone"` is preferred

**Comment by DomT4** [comment]
> If upstream generates a pkg-config file with include/capstone in it, then we should mirror that, I think.

Makes sense if they do, Aye. I just presumed the Capstone maintainer was a-okay with the PR when he dropped by and commented asking for it to be merged the other day. Potential miscommunication/misunderstanding on that one :smile_cat:.

**Pull request closed without merging**